common/functions-v2/src/scalars/arithmetic.rs

```diff
@@ -0,0 +1,154 @@
+// Copyright 2021 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_expression::types::arithmetics_type::ResultTypeOfBinary;
+use common_expression::types::arithmetics_type::ResultTypeOfUnary;
+use common_expression::types::number::*;
+use common_expression::types::DataType;
+use common_expression::vectorize_with_writer_2_arg;
+use common_expression::with_number_mapped_type;
+use common_expression::FunctionProperty;
+use common_expression::FunctionRegistry;
+use common_expression::NumberDomain;
+
+pub fn register(registry: &mut FunctionRegistry) {
+    registry.register_aliases("+", &["plus", "add"]);
```

Comment:
I think the parser won't accept `+(1, 2)` now, do we need to support it?

Reply:
No, I will remove this op.